core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -103,6 +103,8 @@ object ConsoleConsumer extends Logging {
         consumer.stop()
 
         shutdownLatch.await()
+
+        System.out.println("shutdown_complete")
```

Comment:
Or maybe just log at `trace` level perhaps? Looks like `console_consumer.py` is logging at `TRACE`, so we should pick this up?